ui/src/components/EntityTable/EntityTable.jsx

```diff
@@ -113,14 +132,25 @@ class EntityTable extends Component {
         </thead>
         <tbody className={c({'updating': isLoading})}>
           {result.results !== undefined && result.results.map(entity =>
-            <EntityTableRow key={entity.id}
-                            entity={entity}
-                            location={location}
-                            hideCollection={hideCollection}
-                            showPreview={showPreview}
-                            documentMode={documentMode}
-                            updateSelection={updateSelection}
-                            selection={selection} />
+            <React.Fragment>
+              <EntityTableRow key={entity.id}
+                              entity={entity}
+                              location={location}
+                              hideCollection={hideCollection}
+                              showPreview={showPreview}
+                              documentMode={documentMode}
+                              updateSelection={updateSelection}
+                              selection={selection} />
+              {entity.highlight !== undefined && entity.highlight.map(function(sentence, index){
```

Comment:
I think the highlight could be part of `<EntityTableRow/>` component;

Reply:
Strongly agree.